src/main/java/mike/cli/HelperMethods.java

```diff
@@ -91,13 +107,11 @@ private static void loadRelationships(Classes userClasses, JSONObject javaObj, J
 	// Main save function. Calls saveClasses and saveRelationships
 	public static void save (String filename, String directory, Classes userClasses) throws IOException {
 		File newFile = new File(directory);
-		System.out.println();
-		
 		// If directory given does not exist, put in user.dir
 		if(!newFile.isDirectory())
 		{
 			directory = System.getProperty("user.dir");
-			System.out.println("Directory does not exist.");
+			System.out.println("Directory does not exisit.");
```

Comment:
Looks like should be "exist" not "exisit"

Reply:
Other than this, looks good!

Reply:
I swear I changed that like 3 times. Thanks for the catch. Were you able to pull it and run okay with gradle?

Reply:
Found one thing, I'll try to make a separate comment.